src/test/resources/application.yml

```diff
@@ -1,14 +1,19 @@
 server:
-  port: 8080
+  port: 8081
```

Comment:
Why was this changed?

Reply:
Umm... I think there is something wrong with the test because when you run them from the terminal there was a port collision, haven't investigated yet